### ISSUE TITLE
[Feature] Show SSH connections

### ISSUE THREAD

**Issue by spuder** [IceWhaleTech/CasaOS#1510]
As a user of a Terramaster NAS, I enjoy the security that it provides by warning me when a user `ssh`s into the machine

<img width="561" alt="Screenshot 2023-11-15 at 7 23 45 PM" src="https://github.com/IceWhaleTech/CasaOS/assets/242382/22f56ce2-caf3-4846-a6f8-669b9f5b4ec6">

I would like CasaOs/ZimaOs to have the same transparency of ssh connections and show feedback when ssh connections are active. 

https://youtu.be/J_Um3LfLNUo?si=brgnHJXvxYB3jfVT&t=422

**Comment by ETWang1991** [comment]
Yes, this seems very reassuring, my private data is stored in the NAS
